kymatio/scattering2d/backend/tensorflow_backend.py

```diff
@@ -0,0 +1,176 @@
+# Authors: Edouard Oyallon, Sergey Zagoruyko
+
+import tensorflow as tf
+import numpy as np
+from collections import namedtuple
+
+BACKEND_NAME = 'tensorflow'
+
+def _iscomplex(x):
+    return x.dtype == np.complex64 or x.dtype == np.complex128
+
+def _isreal(x):
+    return x.dtype == np.float32 or x.dtype == np.float64
+
+class Pad(object):
+    def __init__(self, pad_size, input_size, pre_pad=False):
+        """
+            Padding which allows to simultaneously pad in a reflection fashion
+            and map to complex.
+            Parameters
+            ----------
+            pad_size : list of 4 integers
+                size of padding to apply.
+            input_size : list of 2 integers
+                size of the original signal
+            pre_pad : boolean
+                if set to true, then there is no padding, one simply adds the imaginarty part.
+        """
+        self.pre_pad = pre_pad
+        self.pad_size = pad_size
+
+    def __call__(self, x):
+        if self.pre_pad:
+            return x
+        else:
+            paddings = [[0, 0]]*len(x.shape[:-2])
```

Comment:
Wouldn't it make sense to be explicit and just write out ` [[0, 0], [0, 0]]`?

Reply:
I'm ok to keep it this wya.

Reply:
I'd prefer `2 * [[0, 0]]` or `2 * [2 * [0]]`.

Reply:
ok.